.readthedocs.yml

```diff
@@ -8,10 +8,14 @@ python:
     - method: pip
       path: .
 
+sphinx:
+  fail_on_warning: true
+
 build:
-    os: ubuntu-22.04
-    tools:
-        python: "miniconda3-4.7"
-    jobs:
-        pre_build:
-            - python .ci/generate-example-mech.py test/mechs/sandiego.yaml pyrometheus/thermochem_example.py
+  os: ubuntu-22.04
+  tools:
+    python: "miniconda3-4.7"
+  jobs:
+    pre_build:
+      # Generate the thermochem example and reinstall pyrometheus as editable
+      - /home/docs/checkouts/readthedocs.org/user_builds/$READTHEDOCS_PROJECT/conda/$READTHEDOCS_VERSION/bin/python .ci/generate-example-mech.py test/mechs/sandiego.yaml pyrometheus/thermochem_example.py && /home/docs/checkouts/readthedocs.org/user_builds/$READTHEDOCS_PROJECT/conda/$READTHEDOCS_VERSION/bin/python -m pip install --no-cache-dir --editable .
```

Comment:
Is this the best way? I ask because it seems awfully brittle. Would `conda activate` work?

Reply:
This is the only way I found that works. 

Activating the conda env does not seem to make a difference. 

Note that rtd *never* activates the conda env it installs and just runs the python binary with the full path 🤷 (like in this PR).

Reply:
As I discovered when working on #67, `pre_build` for some reason doesn't have the virtualenv activated, while `post_install` does 🤷 (`install` comes before `build` in rtd)